conf/app_publication_request.st

```diff
@@ -0,0 +1,3 @@
+$username$ has requested the publication of a new app containing private tools
+in the $environment$ environment. Please use request ID, $apppublicationrequestid$,
```

Comment:
Should we also include the app name in this message? It might be easier to find the request in Belphegor (that is, if the app name is displayed in the request listing).

Maybe including the list of private tools (and version numbers) could be helpful as well?

Reply:
Ah, yeah, that's probably a good idea. I was thinking of this as mostly a message saying, "poke! Poke! Hey, there's stuff to be processed in the app publication request list. Go do the thing to take care of the stuff." 😆 I was thinking that Belphegor could display the list of private tools.

I'll update the template.

Reply:
LGTM :shipit: